frontend/packages/defi-interfaces/pablo/definitions.ts

```diff
@@ -103,8 +103,6 @@ export default {
       assetId: "u128",
     },
     ComposableTraitsDexStakingRewardPool: "Null",
-    RemoveLiquiditySimulationResult: {
-      assets: "BTreeMap<AssetId, Balance>",
-    },
+    RemoveLiquiditySimulationResult: "BTreeMap<AssetId, Balance>",
```

Comment:
There is no longer a `RemoveLiquiditySimulationResult` type, the rpc just returns the BTreeMap directly; not sure if this typedef is required still

Reply:
This is just an alias now.